### ISSUE TITLE
Clip decay for PPO

### ISSUE THREAD

**Issue by TomVeniat** [ikostrikov/pytorch-a2c-ppo-acktr#157]
Hi,
When looking at the argument parser, there is a [`--use-linear-clip-decay`](https://github.com/ikostrikov/pytorch-a2c-ppo-acktr/blob/05a8852cc2ca035ce792b84f902901a50dde0b18/a2c_ppo_acktr/arguments.py#L66) flag avalable for the PPO algorithm but it seems that the clip decay is applied regardless of this parameter and that it only depends on the `--use-linear-lr-decay`.
https://github.com/ikostrikov/pytorch-a2c-ppo-acktr/blob/05a8852cc2ca035ce792b84f902901a50dde0b18/main.py#L107-L108

If this is the case, I think it would be better to use it in the clip decay condition or to remove it completly, as done in the [baseline implementation](https://github.com/openai/baselines/blob/146bbf886ba533fe08b07e01d1c0356aaf7fcc80/baselines/ppo1/pposgd_simple.py#L100) where they always decay the clip param when decaying the lr.

**Comment by ikostrikov** [comment]
Yes, that's a bug. Thanks for catching it!

In ppo1 learning rate decay is decayed with lr. But not in ppo2 for mujoco:
https://github.com/openai/baselines/blob/master/baselines/ppo2/defaults.py

**Comment by ikostrikov** [comment]
Fixed in https://github.com/ikostrikov/pytorch-a2c-ppo-acktr/commit/d93372839bd62b53fa30988172af4d093702e4d5
